src/app/features/fss-home/fss-home.component.html

```diff
@@ -0,0 +1,10 @@
+<div id="mainContainer" role="main">
+<img class="header-image" src="/assets/images/UKHO-17-AW-0005-Web.jpg" alt=" ADMIRALTY FSS Banner">
+<ukho-card>
+        <p>The ADMIRALTY File Sharing Service allows you to search and download files.</p>
+        <h1>Sign in to access your files</h1>
+        <p class="light">You need to sign in with your account to access the File Share Service.</p> 
+        <ukho-button>Sign in</ukho-button>
+        <p><a href="#" id="fss-createaccount">Create a new account</a></p> 
+</ukho-card>
+</div>
```

Comment:
Is this syntactically correct?

I would have assumed that the main heading for this page is:

<p>The ADMIRALTY File Sharing Service allows you to search and download files.</p>

and therefore should have the H1 tags

while

<h1>Sign in to access your files</h1>

is an instruction and not the main heading (therefore should be a lower level heading or some other tag)

I would assume the code needs to be syntactically correct for screen readers/searchability etc.

Please note I am not familiar with the UKHO style system and do not know what impact my comment may have on the layout/styling.

Reply:
We have discussed this issue and have agreed to accept this code as it is currently, merge into main and review any required changes in a subsequent PBI.